src/wymeditor/editor/image-handler.js

```diff
@@ -519,8 +519,8 @@ WYMeditor.ImageHandler.prototype._resizeImage = function (currentMouseY) {
 
     if (!dimensionsRatio) {
         // in order to prevent dimensions ratio corruption
-        var originalHeight = $img.attr('height');
-        var originalWidth = $img.attr('width');
+        var originalHeight = $img.height();
+        var originalWidth = $img.width();
```

Comment:
Yes. I should have been surprised that it actually worked.

Initially, not all images have `height` and `width` attributes. But it did work with those images, nonetheless.

Inside PolicyStat it doesn't, though. So perhaps it is a jQuery difference.